L1Trigger/CSCTriggerPrimitives/src/CSCAnodeLCTProcessor.cc

```diff
@@ -135,6 +134,9 @@ void CSCAnodeLCTProcessor::setConfigParameters(const CSCDBL1TPParameters* conf)
     dumpConfigParams();
     config_dumped = true;
   }
+  minbx_readout_ = CSCConstants::LCT_CENTRAL_BX - l1a_window_width / 2;
+  maxbx_readout_ = CSCConstants::LCT_CENTRAL_BX + l1a_window_width / 2;
+  assert(minbx_readout_ > 0);
```

Comment:
This assert does not work: `minbx_readout_` is unsigned, and it is above 0 by construction
```suggestion
  assert(l1a_window_width / 2 <= CSCConstants::LCT_CENTRAL_BX);
```

Reply:
Moreover: I don't see this setConfigParameters called anywhere.
Who calls it?
Perhaps, another assert also in the constructor won't hurt

Reply:
It's called by the CSCTriggerPrimitives builders [here](https://github.com/cms-sw/cmssw/blob/master/L1Trigger/CSCTriggerPrimitives/plugins/CSCTriggerPrimitivesProducer.cc#L243) and [here](https://github.com/cms-sw/cmssw/blob/master/L1Trigger/CSCTriggerPrimitives/src/CSCTriggerPrimitivesBuilder.cc#L90).